### PR TITLE
Get version of python for installed library

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by ericcurtin** [comment]
I did another version of this in:

https://github.com/containers/ramalama/pull/60

I don't mind which one we go with

**Comment by ericcurtin** [comment]
It iterates through sys.path until it finds the ramalama directory

**Comment by ericcurtin** [comment]
There isn't really a way of determining guest container python version, which is tricky, I think ubi9 might be 3.9, but I don't know

**Comment by ericcurtin** [comment]
Well you could figure it out by doing a quick "podman run --rm" but that starts to get messy and each podman run degrades start time

**Comment by ericcurtin** [comment]
Each podman run is about 150ms from my timings... Maybe that's not too bad... GPU initialisation can take a seconds sometimes anyway

**Comment by ericcurtin** [comment]
We could end up having a mix of Fedora/UBI/Stream containers also for some GPUs... But only if we really need to

File: ramalama/common.py [hunk 1]
```diff
@@ -9,8 +9,15 @@
...
+def in_container():
+    if os.path.exists("/run/.containerenv") or os.path.exists("/.dockerenv") or os.getenv("container"):
+        return True
+
+    return False
+
+
 def container_manager():
-    if sys.platform == "darwin":
+    if in_container() or sys.platform == "darwin":
```
Comment (ericcurtin) [inline ramalama/common.py:20]: Good idea, avoiding nested containers

File: ramalama.py [hunk 2]
```diff
@@ -32,22 +32,26 @@ def main(args):
...
             conman_args = [conman, "run",
                            "--rm",
                            "-it",
                            "--security-opt=label=disable",
                            f"-v{store}:/var/lib/ramalama",
                            f"-v{home}:{home}",
                            "-v/tmp:/tmp",
-                           f"-v{sys.argv[0]}:/usr/bin/ramalama",
-                           f"-v{wd}:{libpath}",
+                           f"-v{__file__}:/usr/bin/ramalama:ro",
```
Comment (ericcurtin) [inline ramalama.py:53]: Also a good idea to add ro

File: ramalama.py [hunk 3]
```diff
@@ -32,22 +32,26 @@ def main(args):
         if host != port:
             port = host.rsplit(':', 1)[1]
 
+        syspath = '/usr/share/ramalama/python'
```
Comment (ericcurtin) [inline ramalama.py:35]: Clever :)

I hope this works and we can exec this before we search for:

```
import errno
import os
from pathlib import Path
import subprocess
import ramalama
import sys
```

**Comment by rhatdan** [comment]
Ok lets go with this one.

File: ramalama.py (hunk 3 above)
Comment (ericcurtin) [reply ramalama.py:35]: We could always write a wrapper python script that did the equivalent of:

```
#!/bin/sh

export PYTHONPATH="/usr/share/ramalama/python:${PYTHONPATH}"
exec ramalama
```

that has basically no imports

**Pull request merged**